packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -316,19 +316,8 @@ class _CodeViewState extends State<CodeView>
                         Expanded(
                           child: LayoutBuilder(
                             builder: (context, constraints) {
-                              // Find the longest line to measure file width:
-                              int longestLength = 0;
-                              TextSpan longestLine;
-                              for (var line in lines) {
-                                final int currentLength =
-                                    line.toPlainText().length;
-                                if (currentLength > longestLength) {
-                                  longestLength = currentLength;
-                                  longestLine = line;
-                                }
-                              }
-                              final double fileWidth =
-                                  calculateTextSpanWidth(longestLine);
+                              final double fileWidth = calculateTextSpanWidth(
+                                  findLongestTextSpan(lines));
```

Comment:
nit: trailing comma

Reply:
Done! OOC, what cases do we need trailing commas? All function parameters?

Reply:
general rule is always* for flutter build methods (*except some cases like a one line SizedBox or similar or a widget with a single unnamed param), and then whenever a function call or method signature has to break to a new line.

Reply:
Ah got it, thanks!